src/util/worker/mod.rs

```diff
@@ -321,6 +387,30 @@ impl<T: Display + Send + 'static> Worker<T> {
         Ok(())
     }
 
+    pub fn start_with_timer<R, U>(&mut self, runner: R, timer: Timer<U>) -> Result<(), io::Error>
```

Comment:
Maybe we can merge this with `start` via a default `Runner` and default Timer like `Timer<()>`.

Reply:
+1

`start` and `start_with_timer` nearly use the same codes.

Reply:
I have tried this:
```rust
struct DefaultRunner<T: Display, R: BatchRunnable>(runner: R);
impl<T: Display, R: BatchRunnable<T>> BatchRunnable<T> for DefaultRunner<T, R> {
    // 3 methods.
}
```
But it conflicts with `impl<T, R: Runable> BatchRunnable<T> for R`.

Any solution for this?